src/ingest-pipeline/airflow/dags/rebuild_metadata.py

```diff
@@ -0,0 +1,198 @@
+import os
+import yaml
+from pprint import pprint
+
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+from airflow.exceptions import AirflowException
+from airflow.configuration import conf as airflow_conf
+from datetime import datetime, timedelta
+
+from utils import (
+    HMDAG,
+    get_queue_resource,
+    get_preserve_scratch_resource,
+    create_dataset_state_error_callback,
+    pythonop_md_consistency_tests,
+    make_send_status_msg_function,
+    get_tmp_dir_path,
+    localized_assert_json_matches_schema as assert_json_matches_schema,
+    pythonop_get_dataset_state,
+    encrypt_tok,
+)
+
+from hubmap_operators.common_operators import (
+    CreateTmpDirOperator,
+    CleanupTmpDirOperator,
+)
+
+
+def get_uuid_for_error(**kwargs):
+    """
+    Return the uuid for the derived dataset if it exists, and of the parent dataset otherwise.
+    """
+    return None
+
+
+def get_dataset_uuid(**kwargs):
+    return kwargs['dag_run'].conf['uuid']
+
+
+def get_dataset_lz_path(**kwargs):
+    ctx = kwargs['dag_run'].conf
+    return ctx['lz_path']
+
+
+default_args = {
+    'owner': 'hubmap',
+    'depends_on_past': False,
+    'start_date': datetime(2019, 1, 1),
+    'email': ['joel.welling@gmail.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=1),
+    'xcom_push': True,
+    'queue': get_queue_resource('launch_multi_analysis'),
```

Comment:
Did you mean to leave this pointed at launch_multi_analysis?  Probably OK but I wanted to make sure it's intentional, since it violates the usual pattern.

Reply:
Good catch, I've updated to reflect the correct DAG.